### ISSUE TITLE
code is included in production builds

### ISSUE THREAD

**Issue by ctcpip** [ember-a11y/ember-a11y-testing#108]
Is it possible to exclude this code from being included when running `ember b --environment=production` ?

**Comment by rwjblue** [comment]
Hmm, that should already be the default behavior...

**Comment by ctcpip** [comment]
_"should"_ being the key word there :)

![screen shot 2018-09-14 at 3 03 13 pm](https://user-images.githubusercontent.com/15326526/45572609-58f19080-b82f-11e8-9df2-454d0e11849f.png)

**Comment by rwjblue** [comment]
D'oh! Looks like we need [this kind of thing](https://github.com/ember-a11y/ember-a11y-testing/blob/750c2e316c4feed66b182ded3bdd6f1752246388/index.js#L54-L66) for `treeForAddon`...

**Comment by ctcpip** [comment]
```
"addon-tree-output/ember-a11y-testing/utils/is-background-replaced-element.js":   1264,
--
"addon-tree-output/ember-a11y-testing/utils/violations-helper.js":   2892, |  
```

**Comment by drewlee** [comment]
I have a fix for this in PR #134

**Comment by drewlee** [comment]
Fixed w/PR #134. Will publish a new release sometime before Monday.
